src/pages/Setup/SetupScreenRenderer.tsx

```diff
@@ -81,7 +80,7 @@ export default function SetupScreenRenderer() {
       return (
         <SyncScreen data={state.syncData} callback={callbacks.onSyncScreen} />
       );
-    case Screen.WAIT:
+    // case Screen.WAIT: UNUSED
```

Comment:
It's not unused, it displays the wait screen :)

Reply:
https://biomejs.dev/linter/rules/no-useless-switch-case/

Yes, but it will use the default, no? If yes we do not need to explicit mention it here, right?